handlers/main.yml

```diff
@@ -6,3 +6,11 @@
     daemon_reload: yes
     state: restarted
     name: "{{ service_name }}"
+  when: ansible_service_mgr == "systemd"
```

Comment:
I guess nowadays the service action plugin could deal with both tasks, so we could have only one task, service: with name {{ service_name }} and daemon_reload: "{{ ansible_service_mgr == 'systemd' | ternary('yes',omit) }}" with no conditionals on the task.

Reply:
Sounds good, I'll update.